pandas_gbq/gbq.py

```diff
@@ -7,6 +7,7 @@
 from distutils.version import StrictVersion
 from time import sleep
 
+import google
```

Comment:
`google` is a namespace package for many different packages. Could you import `google.auth` instead?

Reply:
Also, we don't tend to import our direct dependencies here, since we want the code for imports checking to run before crashing on a failed import.